scripts/build.js

```diff
@@ -8,28 +8,29 @@ const buildConfig = {
   src: 'src',
   dist: 'lib',
   componentsWithStylesheet: [
+    'action-group',
     'body',
     'button',
     'data-table',
+    'doormat',
+    'form',
+    'form-grid',
     'grid',
     'icon',
-    'introduction',
-    'input-group',
+    'image',
+    'info-tile',
```

Comment:
In je story gebruik je geen styles, maar enkel de styling van de component zelf. Moet de css file dan worden aangeboden aan de teams?